src/services/mockData.ts

```diff
@@ -0,0 +1,51 @@
+import Notification, { NotificationState } from '../types/Notification'
+import PatientRecord from '../types/PatientRecord'
+
+const notifications: Notification[] = [
+  {
+    id: '123',
+    type: 'Threshold',
+    created: 1605112212,
+    names: 'Bob Smith',
+    patient_id: '0x5a',
+    state: NotificationState.New,
+    golden_id: '0x45',
+    score: 0.5
+  }
+]
+
+const patientRecord: PatientRecord = {
+  auxId: '0x66',
+  type: 'Current',
```

Comment:
```suggestion
  type: 'Candidate',
```
Should this be candidate?

Reply:
This one is definitely current as it is the current patient. Candidates are potential golden records to link to